src/platform/plugins/shared/dashboard/public/dashboard_container/component/grid/use_layout_styles.tsx

```diff
@@ -55,7 +55,7 @@ export const useLayoutStyles = () => {
       }
 
       .kbnGridPanel--resizeHandle {
-        z-index: ${euiTheme.levels.mask};
+        z-index: ${euiTheme.levels.maskBelowHeader};
```

Comment:
@Heenawter I added a little fix for the resizeHandle to show on header when scrolling (see the resize handle on the kql input). I assumed it's not worth its own issue. Hopefully it doesn't introduce anything new 🤞🏼 
<img width="918" alt="Screenshot 2025-01-24 at 15 36 21" src="https://github.com/user-attachments/assets/334cd9dd-c232-4e23-b9b6-dad65d565047" />

Reply:
I actually am already fixing this in https://github.com/elastic/kibana/pull/207275 (same exact fix, haha!) but it relies on some of my other work on that PR so that the resize handle doesn't disappear on hover. Here's what happens on your PR:

![Jan-24-2025 08-48-45](https://github.com/user-attachments/assets/5bad5c0d-5b37-4722-b688-cae8eeb5490e)

Reply:
perfect, I've reverted 👌🏼